packages/client-sdk-nodejs/test/unit/cache-client.test.ts

```diff
@@ -32,7 +32,9 @@ describe('CacheClient', () => {
     const invalidCacheNames = ['', '    '];
     const momento = await CacheClient.create({
       configuration: configuration,
-      credentialProvider: credentialProvider,
+      credentialProvider: CredentialProvider.fromEnvironmentVariable({
+        environmentVariableName: 'TEST_AUTH_TOKEN',
```

Comment:
why is this changing?

Reply:
This test used to exercise that we get a 4xx if the cache name is invalid. However, it passed an invalid auth token to the client construction. Previously that returned an invalid connection but never hard failed. The test was happy as it could still  call create cache with invalid name and get a 4xx even before the connection was used. Now that the connection had failed to establish, it won't move on from that hard failure. So i just fixed the token here as the test's objective was to assert 4xx on invalid cache names.

Reply:
but this test is a unit test, so it's not supposed to make any network requests. so you shouldn't need a real token here. if the functionality that you want to test requires a real token then we should move it to an int test.